sources/Outputs/src/Par.cpp

```diff
@@ -86,6 +86,8 @@ Par::updateSignalNGenerator(boost::shared_ptr<parameters::ParametersSet> set) {
   set->addReference(helper::buildReference("generator_Q0Pu", "q_pu", "DOUBLE"));
   set->addReference(helper::buildReference("generator_U0Pu", "v_pu", "DOUBLE"));
   set->addReference(helper::buildReference("generator_UPhase0", "angle_pu", "DOUBLE"));
+  set->addReference(helper::buildReference("generator_Pref0Pu", "targetP_pu", "DOUBLE"));
```

Comment:
Does it have an impact on results of the simulation (MAIN tests) ? If not add a main test that is relevant for these changes

Reply:
In the result TestPar.par the line
`<reference type="DOUBLE" name="generator_Pref0Pu" origData="IIDM" origName="targetP_pu"/>`
was added, is it enough ?

Reply:
Uref0Pu should be there too, I think

Reply:
Yes it is also here
`<reference type="DOUBLE" name="generator_Uref0Pu" origData="IIDM" origName="targetV_pu"/>`

Reply:
I was talking about the IIDM output for the whole simulation, performed in MAIN tests. I don't know exactly whet is the rational for the modification but if it corresponds to a functionnality, it should be checked in unit test and an TNR (MAIN tests) should cover it.

Reply:
@marcochiaramello Does this PR introduces change that are worth to be tested in an integration test ? That's what florent meant in these comments.
If so, could you provide an iidm that could be used for this test ?

Reply:
Was this point solved?

Reply:
In my memories yes, @PerninGuillaume do you confirm?

Reply:
Yes